src/Sylius/Component/User/Tests/Security/Checker/EnabledUserCheckerTest.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Tests\Sylius\Component\User\Security\Checker;
+
+use PHPUnit\Framework\MockObject\MockObject;
+use PHPUnit\Framework\TestCase;
+use Sylius\Component\User\Model\UserInterface;
+use Sylius\Component\User\Security\Checker\EnabledUserChecker;
+use Symfony\Component\Security\Core\Exception\DisabledException;
+
+final class EnabledUserCheckerTest extends TestCase
+{
+    /** @var UserInterface&MockObject */
+    private MockObject $userMock;
+
+    private EnabledUserChecker $userChecker;
+
+    protected function setUp(): void
+    {
+        $this->userMock = $this->createMock(UserInterface::class);
```

Comment:
I would be for not using `Mock` suffix
```suggestion
        $this->user = $this->createMock(UserInterface::class);
```